### ISSUE TITLE
A Couple of Mass Mindswap Issues

### ISSUE THREAD

**Issue by AnonymousNow** [tgstation/tgstation#47301]
Some of these may or may not be intended, but:

- Mass Mindswap can choose critical people as targets
- Mass Mindswap can choose people not on the station Z-level as targets
- Mass Mindswap can eject you from an exosuit (presumably because it knocks you down).
- Ghosting as a person who is mindswapped will kill your original body (likely because the other person's ghost vacated the body, were I to guess). Alternatively, it will send you to your original body, which seems to have spontaneously died.

Obviously I haven't been able to replicate these yet, but perhaps someone with buttons can.

Sybil 121960

**Comment by kriskog** [comment]
This event is shitty and should just be removed imo. There are few things as suicide-inducing as mass mindswap, it even contends with donutstation.

**Comment by Ty-the-Smonk** [comment]
> This event is shitty and should just be removed imo. There are few things as suicide-inducing as mass mindswap, it even contends with donutstation.

It sure is a wAcKY wiz event

**Comment by AnonymousNow** [comment]
Honestly, it's fun every so often, but it just has a few hiccups to iron out.

**Comment by carlarctg** [comment]
i disagree, it's objectively terrible to have a fun round as captain or head or anyrhing ruined due to mindswap putting you in a miner that was fighting fauna's body

**Comment by AnonymousNow** [comment]
I think that'd be covered by making mass mindswap not mess with people off the station Z-level.

**Comment by Mooshimi** [comment]
This event may have been removed, as I can't find it.
